content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -71,74 +142,62 @@ front-proxy-ca          Dec 28, 2029 23:36 UTC   9y              no
 
 また、証明書が外部管理されている場合、kubeadmはユーザーに通知します。この場合、ユーザーは証明書の更新を手動または他のツールを使用して管理する必要があります。
 
-{{< warning >}}
-`kubeadm`は外部CAによって署名された証明書を管理することができません。
-{{< /warning >}}
-
-{{< note >}}
-kubeadmは`/var/lib/kubelet/pki`以下にあるローテート可能な証明書でkubeletの[証明書の自動更新](/docs/task/tls/certificate-rotation/)を構成するので`kubelet.conf`は上記のリストに含まれません。
-
+kubeadmは`/var/lib/kubelet/pki`以下にある更新可能な証明書でkubeletの[証明書の自動更新](/docs/tasks/tls/certificate-rotation/)を構成するので`kubelet.conf`は上記のリストに含まれません。
```

Comment:
```suggestion
kubeadmは`/var/lib/kubelet/pki`配下にあるローテーション可能な証明書でkubeletの[証明書の自動更新](/docs/tasks/tls/certificate-rotation/)を構成するため、`kubelet.conf`は上記のリストに含まれません。
```
- 以下 -> 配下
- ～ので -> ～ため(ガイドラインより)
- 読点を追加

> The `kubelet.conf` configuration file is not included in the list above because kubeadm configures kubelet for [automatic certificate renewal](https://kubernetes.io/docs/tasks/tls/certificate-rotation/) with rotatable certificates under `/var/lib/kubelet/pki`.

それと、このページでは証明書に対して、renewalとrotationの2つの表現が使われています。
どちらも更新することを意味していますが、rotationは定期的なサイクルで新しくするという意味があるため、日本語では更新よりもローテーションとしておくのが適切ではないかと思いました。
Kubernetesに限らず、一般的な証明書に関するドキュメントでも、ローテーションという言葉はよく使われているようでした。
これ以降も、該当する箇所はコメントしておきました。